### PR TITLE
Request for Harbor Graduation Review

### PR DESCRIPTION
**Pull request by michmike**
SIG-Runtime Graduation review request of Harbor as per https://github.com/cncf/toc/pull/311

### CODE REVIEW

File: recommendations/graduated/harbor.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+https://github.com/goharbor
+
+**_Link to Tech-DD_**
+
+https://docs.google.com/document/d/15gX7EeeXQThEvVMGpL-0a1mOwGuByLtMfvXNJaKT0A0/edit?usp=sharing
+
+**_Link to SIG-Runtime Tech-DD for Harbor Graduation_**
+
+http://bit.ly/harbor-graduation-dd
```
Comment (k82cn) [inline recommendations/graduated/harbor.md]: this doc is still WIP?

Comment (michmike) [reply recommendations/graduated/harbor.md]: hi @k28cn, this is the PR that will do the work that goes into that DD doc (essentially verifying compliance with CNCF graduation criteria).

**Comment by raravena80** [comment]
@michmike I was hoping to answer the graduation requirements from your side and we as co-chairs and tech-leads would verify the answers based on the DD doc, github repo, etc. Is that doable? Thanks.

**Comment by michmike** [comment]
> @michmike I was hoping to answer the graduation requirements from your side and we as co-chairs and tech-leads would verify the answers based on the DD doc, github repo, etc. Is that doable? Thanks.

added. thanks

File: recommendations/graduated/harbor.md (hunk 1 above)
Comment (k82cn) [reply recommendations/graduated/harbor.md]: Great! I'll share my points after went through the related doc :)

File: recommendations/graduated/harbor.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+* [x] [Yes](https://github.com/goharbor/harbor/blob/master/ADOPTERS.md)
+10. Any other substantive concerns expressed by SIGs or TOC members 
+* None
+
+### Summary of what was reviewed, and by who
+* Reviewer: Michael Michael (Harbor Maintainer, Director of Product Management at VMware)
+* Approver: Ricardo Aravena (CNCF SIG-Runtime Chair)
+
+### List of any substantive concerns
+* None
```
Comment (raravena80) [inline recommendations/graduated/harbor.md]: @michmike I reviewed the DD document and made a few comments. This LGTM as long as all the comments from the community in the DD doc are addressed. Thanks!

Comment (michmike) [reply recommendations/graduated/harbor.md]: @raravena80 i replied to all your comments (as well as the comments from the community), addressed them, and closed them. the document is now up to date.

**Pull request merged**